### PR TITLE
Add Autoscaler methods

### PR DESCRIPTION
**Pull request by EtienneM**
Related to Scalingo/autoscaler-service#1

### CODE REVIEW

**Comment by Soulou** [comment]
Please rebase

File: autoscalers.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	AutoscalerAdd(app string, params AutoscalerAddParams) (*Autoscaler, error)
+	AutoscalerRemove(app string, id string) error
+}
+
+var _ AutoscalersService = (*Client)(nil)
+
+type Autoscaler struct {
+	ID            string  `json:"id"`
+	ContainerType string  `json:"container_type"`
+	Metric        string  `json:"metric"`
```
Comment (johnsudaar) [inline autoscalers.go]: Would be nice to habe a list of all the available metrics here.

Comment (EtienneM) [reply autoscalers.go]: I don't really get what you mean here

Comment (johnsudaar) [reply autoscalers.go]: Would be nice if you could add constants with all the possible metrics.

Comment (EtienneM) [reply autoscalers.go]: We actually have an endpoint in the alerter which lists all the metrics (cf. Scalingo/scalingo-alerter-service#54)

**Pull request merged**